Cargo.toml

```diff
@@ -29,31 +29,31 @@ with-nexmark = [
 [dependencies]
 num = "0.4.0"
 anyhow = "1.0.57"
-size-of = "0.1.1"
+size-of = { version = "0.1.2", features = ["time-std"]}
 petgraph = "0.6.0"
 typedmap = { version = "0.3.0", features = ["dashmap"] }
 crossbeam-utils = "0.8.6"
 once_cell = "1.9.0"
 priority-queue = "1.2.1"
 hashbrown = "0.12.0"
 csv = { version = "1.1", optional = true }
-serde = { version = "1.0", optional = true }
+serde = { version = "1.0", features = ["derive"], optional = true }
 impl-trait-for-tuples = "0.2"
 textwrap = "0.15.0"
-fxhash = "0.2"
-ordered-float = "3.0.0"
+ordered-float = { version = "3.0.0", features = ["serde"]}
 arcstr = { version = "1.1.4", optional = true }
 rust_decimal = { version = "1.26.1", optional = true }
 regex = { version = "1.6.0", optional = true }
 time = { version = "0.3.14", features = ["formatting"], optional = true }
 paste = { version = "1.0.9", optional = true }
 bitvec = "1.0.1"
+twox-hash = "1.6.3"
```

Comment:
I tend to prefer [`xxhash-rust`](https://docs.rs/xxhash-rust) since it utilizes simd and is a really solid implementation